atomata/VClassroom_Proto_VSolar_ApparatusUPackageDevelopment
# Webgl communication bundle refactor

This branch adds the pluging for the webgl browser communication. Also, a new scene and container code are created that loads the apparatus as asset bundles from azure instead of addressables . 
To test: 

 build the scene appratuscontainer_exampleassetbundlecloud and try the load and trigger buttons. the assets are all loaded from the azure . 

To test webgl/browser communication : 

after building the scene, got to index.html and add this to store the unity instance in uin variable : 
 var uin = null;
  script.onload = () => {
    createUnityInstance(canvas, config, (progress) => {...}).then((unityInstance) => {
      uin = unityInstance;
      …

Then click on cube button to load the apparatus. Then in the browser console , type this command :  uin.SendMessage("Container",”VoidTrigger", "gracious-cube/gracious-cube/delta-sphere/wobble-sphere/wobble-sphere/animations@Wobble") to make the spehere wobble. 

A message should be printed on the browser console : " The void trigger is handled successfully"
